{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","driverError":{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","file":"nbtinsert.c","length":198,"line":"656","name":"error","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","table":"user"},"file":"nbtinsert.c","length":198,"level":"error","line":"656","message":"duplicate key value violates unique constraint \"user_username_key\" duplicate key value violates unique constraint \"user_username_key\"","parameters":["gdo","c58f257617be7e34c4d731f6c4f3dfeb05a8257d3063a77f873d3668763abc7d.b37be9c7242a1b9e",202,2],"query":"INSERT INTO \"sec\".\"user\"(\"username\", \"password\", \"passcode\", \"is_new_password\", \"is_active\", \"form_id\", \"org_id\", \"created_by\", \"updated_by\", \"created_at\", \"updated_at\", \"contact_id\", \"role_id\") VALUES ($1, $2, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, $3, $4) RETURNING \"id\", \"is_new_password\", \"is_active\", \"created_by\", \"created_at\", \"updated_at\"","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","stack":"QueryFailedError: duplicate key value violates unique constraint \"user_username_key\"\n    at QueryFailedError.TypeORMError [as constructor] (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\TypeORMError.ts:7:9)\n    at new QueryFailedError (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\QueryFailedError.ts:9:9)\n    at PostgresQueryRunner.<anonymous> (C:\\CMR\\gdo\\checkin\\nestjs\\src\\driver\\postgres\\PostgresQueryRunner.ts:247:19)\n    at step (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:143:27)\n    at Object.throw (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:124:57)\n    at rejected (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:115:69)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","table":"user"}
{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","driverError":{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","file":"nbtinsert.c","length":198,"line":"656","name":"error","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","table":"user"},"file":"nbtinsert.c","length":198,"level":"error","line":"656","message":"duplicate key value violates unique constraint \"user_username_key\" duplicate key value violates unique constraint \"user_username_key\"","parameters":["gdo","407ae74127c1dae650570fefe240d786642d3ca59b63d1d581af1211b072209b.ca112fa20e53d633",203,2],"query":"INSERT INTO \"sec\".\"user\"(\"username\", \"password\", \"passcode\", \"is_new_password\", \"is_active\", \"form_id\", \"org_id\", \"created_by\", \"updated_by\", \"created_at\", \"updated_at\", \"contact_id\", \"role_id\") VALUES ($1, $2, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, $3, $4) RETURNING \"id\", \"is_new_password\", \"is_active\", \"created_by\", \"created_at\", \"updated_at\"","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","stack":"QueryFailedError: duplicate key value violates unique constraint \"user_username_key\"\n    at QueryFailedError.TypeORMError [as constructor] (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\TypeORMError.ts:7:9)\n    at new QueryFailedError (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\QueryFailedError.ts:9:9)\n    at PostgresQueryRunner.<anonymous> (C:\\CMR\\gdo\\checkin\\nestjs\\src\\driver\\postgres\\PostgresQueryRunner.ts:247:19)\n    at step (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:143:27)\n    at Object.throw (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:124:57)\n    at rejected (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:115:69)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","table":"user"}
{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","driverError":{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","file":"nbtinsert.c","length":198,"line":"656","name":"error","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","table":"user"},"file":"nbtinsert.c","length":198,"level":"error","line":"656","message":"duplicate key value violates unique constraint \"user_username_key\" duplicate key value violates unique constraint \"user_username_key\"","parameters":["gdo","b75568e67fcc8438066ba994a9f3e9cf08e4732b3aa54a071dc386040444682d.c32f206ad8b9601c",204,2],"query":"INSERT INTO \"sec\".\"user\"(\"username\", \"password\", \"passcode\", \"is_new_password\", \"is_active\", \"form_id\", \"org_id\", \"created_by\", \"updated_by\", \"created_at\", \"updated_at\", \"contact_id\", \"role_id\") VALUES ($1, $2, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, $3, $4) RETURNING \"id\", \"is_new_password\", \"is_active\", \"created_by\", \"created_at\", \"updated_at\"","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","stack":"QueryFailedError: duplicate key value violates unique constraint \"user_username_key\"\n    at QueryFailedError.TypeORMError [as constructor] (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\TypeORMError.ts:7:9)\n    at new QueryFailedError (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\QueryFailedError.ts:9:9)\n    at PostgresQueryRunner.<anonymous> (C:\\CMR\\gdo\\checkin\\nestjs\\src\\driver\\postgres\\PostgresQueryRunner.ts:247:19)\n    at step (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:143:27)\n    at Object.throw (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:124:57)\n    at rejected (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:115:69)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","table":"user"}
{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","driverError":{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","file":"nbtinsert.c","length":198,"line":"656","name":"error","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","table":"user"},"file":"nbtinsert.c","length":198,"level":"error","line":"656","message":"duplicate key value violates unique constraint \"user_username_key\" duplicate key value violates unique constraint \"user_username_key\"","parameters":["gdo","2aae5420de364cecbfaa4a9276207b86071212de3b64fc5648d545bd2a257842.be6e6f762a50c02f",205,2],"query":"INSERT INTO \"sec\".\"user\"(\"username\", \"password\", \"passcode\", \"is_new_password\", \"is_active\", \"form_id\", \"org_id\", \"created_by\", \"updated_by\", \"created_at\", \"updated_at\", \"contact_id\", \"role_id\") VALUES ($1, $2, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, $3, $4) RETURNING \"id\", \"is_new_password\", \"is_active\", \"created_by\", \"created_at\", \"updated_at\"","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","stack":"QueryFailedError: duplicate key value violates unique constraint \"user_username_key\"\n    at QueryFailedError.TypeORMError [as constructor] (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\TypeORMError.ts:7:9)\n    at new QueryFailedError (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\QueryFailedError.ts:9:9)\n    at PostgresQueryRunner.<anonymous> (C:\\CMR\\gdo\\checkin\\nestjs\\src\\driver\\postgres\\PostgresQueryRunner.ts:247:19)\n    at step (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:143:27)\n    at Object.throw (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:124:57)\n    at rejected (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:115:69)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","table":"user"}
{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","driverError":{"code":"23505","constraint":"user_username_key","detail":"Key (username)=(gdo) already exists.","file":"nbtinsert.c","length":198,"line":"656","name":"error","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","table":"user"},"file":"nbtinsert.c","length":198,"level":"error","line":"656","message":"duplicate key value violates unique constraint \"user_username_key\" duplicate key value violates unique constraint \"user_username_key\"","parameters":["gdo","0c6b3d53d7d4b27130f24d400e9dd6f27debefe8a2cd3b73d68990b6d97a3dc4.7c78876a83b1e08b",206,2],"query":"INSERT INTO \"sec\".\"user\"(\"username\", \"password\", \"passcode\", \"is_new_password\", \"is_active\", \"form_id\", \"org_id\", \"created_by\", \"updated_by\", \"created_at\", \"updated_at\", \"contact_id\", \"role_id\") VALUES ($1, $2, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, $3, $4) RETURNING \"id\", \"is_new_password\", \"is_active\", \"created_by\", \"created_at\", \"updated_at\"","routine":"_bt_check_unique","schema":"sec","severity":"ERROR","stack":"QueryFailedError: duplicate key value violates unique constraint \"user_username_key\"\n    at QueryFailedError.TypeORMError [as constructor] (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\TypeORMError.ts:7:9)\n    at new QueryFailedError (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\QueryFailedError.ts:9:9)\n    at PostgresQueryRunner.<anonymous> (C:\\CMR\\gdo\\checkin\\nestjs\\src\\driver\\postgres\\PostgresQueryRunner.ts:247:19)\n    at step (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:143:27)\n    at Object.throw (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:124:57)\n    at rejected (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:115:69)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","table":"user"}
{"level":"error","message":"Cannot create property 'Symbol(level)' on string 'duplicate key value violates unique constraint \"user_username_key\"'"}
{"level":"error","message":"Cannot create property 'Symbol(level)' on string 'duplicate key value violates unique constraint \"user_username_key\"'"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"main\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No metadata for \"Session\" was found."}
{"level":"error","message":"No metadata for \"Session\" was found."}
{"level":"error","message":"No metadata for \"Session\" was found."}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"secret option required for sessions"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No metadata for \"Session\" was found."}
{"level":"error","message":"No metadata for \"Session\" was found."}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"code":"42P01","driverError":{"code":"42P01","file":"d:\\pginstaller.auto\\postgres.windows-x64\\src\\backend\\parser\\parse_relation.c","length":171,"line":"1159","name":"error","position":"13","routine":"parserOpenTable","severity":"ERROR"},"file":"d:\\pginstaller.auto\\postgres.windows-x64\\src\\backend\\parser\\parse_relation.c","length":171,"level":"error","line":"1159","message":"relation \"org.calendar\" does not exist relation \"org.calendar\" does not exist","parameters":["gdo"],"position":"13","query":"INSERT INTO \"org\".\"calendar\"(\"name\", \"location_id\", \"org_id\") VALUES ($1, DEFAULT, DEFAULT) RETURNING \"id\"","routine":"parserOpenTable","severity":"ERROR","stack":"QueryFailedError: relation \"org.calendar\" does not exist\n    at QueryFailedError.TypeORMError [as constructor] (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\TypeORMError.ts:7:9)\n    at new QueryFailedError (C:\\CMR\\gdo\\checkin\\nestjs\\src\\error\\QueryFailedError.ts:9:9)\n    at PostgresQueryRunner.<anonymous> (C:\\CMR\\gdo\\checkin\\nestjs\\src\\driver\\postgres\\PostgresQueryRunner.ts:247:19)\n    at step (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:143:27)\n    at Object.throw (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:124:57)\n    at rejected (C:\\CMR\\gdo\\checkin\\nestjs\\node_modules\\tslib\\tslib.js:115:69)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"column \"mapping\" contains null values"}
{"level":"error","message":"constraint \"PK_f55da76ac1c3ac420f444d2ff11\" of relation \"session\" does not exist"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"constraint \"PK_f55da76ac1c3ac420f444d2ff11\" of relation \"session\" does not exist"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
{"level":"error","message":"No repository for \"Session\" was found. Looks like this entity is not registered in current \"default\" connection?"}
